Formula/openssl@1.1.rb

```diff
@@ -23,14 +23,9 @@ class OpensslAT11 < Formula
 
   keg_only :shadowed_by_macos, "macOS provides LibreSSL"
 
-  on_linux do
-    resource "cacert" do
-      # homepage "http://curl.haxx.se/docs/caextract.html"
-      url "https://curl.haxx.se/ca/cacert-2020-01-01.pem"
-      mirror "https://gist.githubusercontent.com/dawidd6/16d94180a019f31fd31bc679365387bc/raw/ef02c78b9d6427585d756528964d18a2b9e318f7/cacert-2020-01-01.pem"
-      sha256 "adf770dfd574a0d6026bfaa270cb6879b063957177a991d453ff1d302c02081f"
-    end
+  depends_on "ca-certificates"
 
+  on_linux do
```

Comment:
This can probably be dealt with in a separate PR, but I wonder about these `resource`s. Does OpenSSL really need these at runtime? These look like build dependencies to me.

Reply:
No, but I think there's some dependents that are. This should be moved elsewhere but that's going to be out of scope for this PR.